types/next-auth/client.d.ts

```diff
@@ -50,10 +50,16 @@ declare function signin(
     provider?: string,
     data?: GenericObject & {
         callbackUrl?: string;
+        redirect?: boolean;
     },
 ): Promise<void>;
 declare const signIn: typeof signin;
-declare function signout(data?: { callbackUrl?: string }): Promise<void>;
+declare function signout(
+    data?: { 
```

Comment:
There's a trailing white-space on this line 📐 , which you'll need to remove for the lint process [not to complain on the CI](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/51477/checks?check_run_id=1999784269) 👍🏼